app/templates/events/event_list.html

```diff
@@ -67,7 +67,16 @@ <h1 class="text-center mt-5">Events List for {{selectedTerm.description}}</h1>
           <tr>
             <td> <a href="/eventsList/{{event.id}}/edit" class="link-primary">{{event.name}}</a></td>
             <td>{{event.startDate.strftime('%m-%d-%Y')}}</td>
-            <td>{{event.timeStart.strftime('%I:%M %p')}}-{{event.timeEnd.strftime('%I:%M %p')}}</td>
+            <td>{{event.startDate.strftime('%m-%d-%Y')}}</td>
+            {% set startTime = event.timeStart.strftime('%I:%M %p') %}
+            {% set endTime = event.timeEnd.strftime('%I:%M %p') %}
+            {% if startTime[0] == '0' %}
+                {% set startTime = startTime[1:] %}
+            {% endif %}
+            {% if endTime[0] == '0' %}
+                {% set endTime = endTime[1:] %}
+            {% endif %}
```

Comment:
You can remove the leading zeroes in the format string, with `%-I` instead of `%I`

Reply:
resolved